src/asyncOperationManagerUtils.js

```diff
@@ -61,7 +61,18 @@ const registerAsyncOperationDescriptors = (asyncOperationDescriptors, ...otherDe
   return asyncOperationManagerState.setState(newState);
 };
 
-const getAsyncOperation = (state, descriptorId, params, otherFields) => {
+const invalidateAsyncOperationByKey = (asyncOperationKey, descriptorId) => {
```

Comment:
It seems like we could do `invalidateAsyncOperation = (descriptorId, asyncOperationKey)`

Where asyncOperationKey is optional and targets the specific operation and if left out, invalidates all of the descriptor's operations. That would eliminate the need for `invalidatingOperationsDescriptorIds` and leave the invalidating solely up to the consumer, simplifying this library's role.

Reply:
we could still implement this if it's still desired.